Resources/Prototypes/_Nuclear14/Entities/Mobs/NPCs/pets.yml

```diff
@@ -0,0 +1,17 @@
+#MARK: Haggis
+- type: entity
+  name: Haggis
+  id: N14MobCatShopkeep
```

Comment:
Probably worth changing this to CatHaggis since it's the name people will search for